dojo/1.11/_base.d.ts

```diff
@@ -1275,7 +1275,7 @@ declare namespace dojo {
 			/**
 			 * Copies/adds all properties of one or more sources to dest; returns dest.
 			 */
-			_mixin<T, U>(dest: T, source: U, copyFunc?: (s: any) => any): T & U;
+			mixin<T, U>(dest: T, source: U, copyFunc?: (s: any) => any): T & U;
```

Comment:
[`_mixin` is correctly typed](https://github.com/dojo/dojo/blob/06203f380069d21b48b606577ea3d507cbfd381b/_base/lang.js#L68) but what is missing is:

```typescript
/**
 * Copies/adds all properties of one or more sources to dest; returns dest.
 */
mixin<T, U>(dest: T, sources: U): T & U;
mixin<T, U, V>(dest: T, source1: U, source2: V): T & U & V;
mixin<T, U, V, W>(dest: T, source1: U, source2: V, source3: W): T & U & V & W;
mixin<T, U, V, W, X>(dest: T, source1: U, source2: V, source3: W, source4: X): T & U & V & W & X;
mixin<T, U, V, W, X, Y>(dest: T, source1: U, source2: V, source3: W, source4: X, source5: Y): T & U & V & W & X & Y;
mixin<T, U, V, W, X, Y, Z>(dest: T, source1: U, source2: V, source3: W, source4: X, source5: Y, source6: Z): T & U & V & W & X & Y & Z;
mixin<T, U>(dest: T, ...sources: U[]): T & U;
```